docs/src/pages/[platform]/components/authenticator/customization/index.page.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: Customization
+metaTitle: Customization
+description: Override and customize your Authenticator.
+supportedFrameworks: react|angular|vue
+---
+
+import { Fragment } from '@/components/Fragment';
+
+<Fragment platforms={['flutter']}>
+  {({ platform }) => import(`./customization.routing.${platform}.mdx`)}
+</Fragment>
+
+<Fragment useCommonWebContent platforms={['web']}>
+  {({ platform }) =>
+    import(`./customization.headers-and-footers.${platform}.mdx`)
```

Comment:
I can't comment on these files because they are just re-names, but I think these have headings in them that should be bumped up.

Reply:
I looked into it and bumped all the h3s and h4s up as well. 3fa06580eb09862a439986cc89893f9f241d7ddf